### PR TITLE
display real deleted custom types in changes page

### PR DESCRIPTION
**Pull request by mathieud**
## Checklist before requesting a review
- [ ] I hereby declare my code ready for review.
- [ ] If it is a critical feature, I have added tests.
- [ ] The CI is successful.
- [ ] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: packages/slice-machine/src/modules/availableCustomTypes/types.ts [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
+  remote: CustomTypeSM;
+};
+export type LocalFrontEndCustomType = NewCustomType | SyncedCustomType;
+
+export type DeletedFrontEndCustomType = {
+  local: undefined;
+  remote: CustomTypeSM;
+};
+
+// TODO whats the difference between this and packages/slice-machine/lib/models/common/ModelStatus/compareCustomTypeModels.ts::FrontEndCtModel
```
Comment (danigo99) [inline packages/slice-machine/src/modules/availableCustomTypes/types.ts:18]: Still needed?

File: packages/slice-machine/src/theme.ts [hunk 2]
```diff
@@ -37,6 +37,10 @@ const AppTheme = (): Theme =>
           bg: "#F1EEFE",
           color: "#5842C3",
         },
+        deleted: {
+          bg: "rgba(203, 36, 49, 0.1)",
+          color: "#CB2431",
```
Comment (danigo99) [inline packages/slice-machine/src/theme.ts]: We already have this colour, it's called "danger"

**Review by danigo99 — COMMENTED** [review]
Mostly looks good I think!

File: packages/slice-machine/src/modules/availableCustomTypes/types.ts (hunk 1 above)
Comment (mathieud) [reply packages/slice-machine/src/modules/availableCustomTypes/types.ts:18]: @Heyrows ?

File: packages/slice-machine/src/theme.ts (hunk 2 above)
Comment (mathieud) [reply packages/slice-machine/src/theme.ts]: Fixed

**Review by danigo99 — APPROVED** [review]

**Pull request merged**